samples/drivers/i2s/echo/src/main.c

```diff
@@ -272,6 +272,7 @@ int main(void)
 		return 0;
 	}
 
+	printf("config stream\n");
 	config.word_size = SAMPLE_BIT_WIDTH;
```

Comment:
@hakehuang  do you really want to add this prints into mainline? This will affect everyone one using this sample app.

Reply:
It need to be removed. Thanks for remind